src/test/java/tictactoe/BoardConverterTest.java

```diff
@@ -1,12 +1,13 @@
 package tictactoe;
 
+import org.junit.Ignore;
```

Comment:
That annotation doesn't seem to be used anymore.
It's good to "Organize Imports" from time to time. If I remember correctly the IntelliJ shortcut for that is ctrl+alt+o.